docs/guide/deployer.md

```diff
@@ -284,3 +284,33 @@ const [approvalInfo, clearInfo] = [info.approval, info.clear];
 
 const lsigInfo = await deployer.getDeployedASC("MyLsig");
 ```
+
+## Helper methods
+
+`Deployer` class expose helper methods to improve developer's experience with the framewok. These methods can be also found under the same name and for most of the time with the same interfaces in `myalgowallet-mode`, `web-mode`, `wallectconnect-mode` and `Runtime`. These methods enable users to use `algosdk` types like `Transaction` and `SignedTransaction`. 
+List of the methods:
+- `makeTx` - creates`Transaction` object from `execParams`
+- `signTx` - signes `Transaction` and returns `SignedTransaction` object.
+- `makeAndSignTx` - combines funcnionality of the two methods listed above
+- `sendTxAndWait` - sends `SignedTransaction` and waits for the response that is returned to the user.
+
+### Practical example:
+```ts
+const execParams: AlgoTransferParam = {
+			type: TransactionType.TransferAlgo,
+			sign: SignType.SecretKey,
+			fromAccount: alice,
+			toAccountAddr: bob.addr,
+			amountMicroAlgos: 10000n,
+			payFlags: {},
+		};
+const signature: wtypes.Sign = {
+  sign: SignType.SecretKey,
+  fromAccount: alice,
+};
+const txnParams = mockSuggestedParam;
```

Comment:
`mockSuggestedParam` is not declared above. User won't know what it is.
Will this work here? `algodClient.getTransactionParams().do()`

Reply:
I updated it